languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -453,7 +453,7 @@ though, and has since been slightly modified)-->
         <example correction="faible">C'est le plus <marker>mauvais</marker> score de ce jeu.</example>
       </rule>
     </rulegroup>
-    <rulegroup id="PLUS_MIEUX" name="plus mieux" default="temp_off">
+    <rulegroup id="PLUS_MIEUX" name="plus mieux">
```

Comment:
Could you make the message just a bit nicer? "est un collocation inadéquate" is true, but harsh. "est considérée fautive dans le langage standard" or something like that, maybe?